Dockerfile

```diff
@@ -17,6 +29,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends python3-pip pyt
 # Directory where the app is installed and run.
 WORKDIR /usr/src/app
 
+# Copy SSL certificate from the cert-gen stage
+COPY --from=cert-gen /certs/server.key /usr/src/app/server.key
+COPY --from=cert-gen /certs/server.crt /usr/src/app/server.crt
```

Comment:
Do we need to copy these files, are they required to build the APP?
Maybe they should be copied only in the last state.

Reply:
Good point, if docker allows we may as well copy them from stage 1 to stage 3 directly.

Reply:
I checked it locally and it works, added a suggestion in another comment.

So we can remove these lines if they are not required
```suggestion
```